core/capabilities/webapi/outgoing_connector_handler.go

```diff
@@ -114,10 +133,22 @@ func (c *ConnectorHandler) HandleGatewayMessage(ctx context.Context, gatewayID s
 	}
 }
 
-func (c *ConnectorHandler) Start(ctx context.Context) error {
-	return c.gc.AddHandler([]string{webapicapabilities.MethodWebAPITarget}, c)
+func (c *OutgoingConnectorHandler) Start(ctx context.Context) error {
+	return c.gc.AddHandler([]string{c.method}, c)
 }
 
-func (c *ConnectorHandler) Close() error {
+func (c *OutgoingConnectorHandler) Close() error {
 	return nil
 }
+
+func validMethod(method string) bool {
```

Comment:
nit, maybe this instead:

```suggestion
func validMethod(method string) bool {
	switch method {
	case capabilities.MethodWebAPITarget, capabilities.MethodWebAPITrigger, capabilities.MethodComputeAction:
		return true
	default:
		return false
	}
}
```

Reply:
Thanks, I thought of doing it like this and then opted for the slice + range thinking of more values being added in the future. probably not something to worry now, will do it with the switch as you mentioned and actually remove the `capabilities.MethodWebAPITrigger` given this is the outgoing handler